RecoTracker/LSTCore/interface/alpaka/Constants.h

```diff
@@ -0,0 +1,183 @@
+#ifndef RecoTracker_LSTCore_interface_alpaka_Constants_h
+#define RecoTracker_LSTCore_interface_alpaka_Constants_h
+
+#include <alpaka/alpaka.hpp>
+
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+
+#ifdef CACHE_ALLOC
+#include "HeterogeneousCore/AlpakaInterface/interface/CachedBufAlloc.h"
+#endif
+
+#ifdef ALPAKA_ACC_GPU_CUDA_ENABLED
+#include <cuda_fp16.h>
+#endif
+
+namespace SDL {
+// Half precision wrapper functions.
+#if defined(FP16_Base)
```

Comment:
What are the plans for 16-but floats ?
Would it make sense to use `_Float16` on the cpu, especially in conjunction with AVX-512 support ?

Reply:
it seems like this is a question to alpaka
https://github.com/alpaka-group/alpaka/issues/2004
the response sounds like c++23 support could give some progress

Reply:
I meant if the LST authors plan to keep fp16 functionality in the code or not.
If yes, there should be some way to test it, on some backend.
If not, I would ask to remove the corresponding defines and typdefs.